apps/www/components/LaunchWeek/Ticket/TicketForm.tsx

```diff
@@ -66,27 +66,33 @@ export default function TicketForm({ defaultUsername = '', setTicketGenerationSt
           setPageState('ticket')
 
           // Listen to realtime changes
-          const channel =
-            realtimeChannel ??
-            supabase
-              .channel('changes')
-              .on(
-                'postgres_changes',
-                {
-                  event: 'UPDATE',
-                  schema: 'public',
-                  table: 'lw7_tickets',
-                  filter: `username=eq.${username}`,
-                },
-                (payload) => {
-                  setUserData({
-                    ...payload.new,
-                    golden: !!payload.new.sharedOnTwitter && !!payload.new.sharedOnLinkedIn,
-                  })
-                }
-              )
-              .subscribe()
-          setRealtimeChannel(channel)
+          if (!realtimeChannel && !data.golden) {
+            const channel =
+              realtimeChannel ??
```

Comment:
can remove this right? since we're checking if realtimeChannel is undefined - so it'll definitely be undefined when it enters this block

Reply:
The `useEffect` is triggered every time the `session` changes. That's why we need to check.

Reply:
ahh, you mean the default specifier. yah, you're right, thanks